content/en/database_monitoring/setup_oracle/_index.md

```diff
@@ -24,26 +24,93 @@ The features described on this page are in private beta.
 The following deployment configurations, components, and features are not supported: Oracle Autonomous Database, ASM, and Data Guard.
```

Comment:
Meaghan let me know that ASM is also a Datadog product - to avoid confusion, can we spell out "Automatic Storage Management" here?

Reply:
Has anyone explicitly asked for ASM? If not, we might be better of by leaving out ASM completely. There's no performance monitoring for ASM. It would be only availably check. I think the remark might turn people off who might otherwise interested in the product.

Reply:
It's similar to explicitly stating (for SQL Server) "We don't support deadlock or tempdb". Nobody asks this until they start using the product and then notice the gap when it becomes relevant.

Reply:
Ah ok, that's helpful. It hasn't come up at all from what I can tell, so let's get rid of it